File: java/ModuleTest03/src/main/java/com/adrianpoplesanu/ModuleTest03/Controller/HomeController.java
package com.adrianpoplesanu.ModuleTest03.Controller;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.ResponseBody;

@Controller
public class HomeController {
    @RequestMapping(value = "/", method = RequestMethod.GET)
    @ResponseBody
    public String index(@Value("${test}") String text) {
        return text;
    }
}

File: kotlin/DemoProject1/src/main/kotlin/com/adrianpoplesanu/DemoProject1/controller/HomeController.kt
package com.adrianpoplesanu.DemoProject1.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.ResponseBody

@Controller
class HomeController {
    @GetMapping("/")
    @ResponseBody
    fun index() : String {
        return "aaa"
    }

    @GetMapping("test2")
    @ResponseBody
    fun test2() : String {
        val text = this::class.java.classLoader.getResource("test.csv").readText()
        return text
    }
}